pkgs/tools/security/sshocker/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sshocker";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "lima-vm";
+    repo = "sshocker";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-u/H9X0YbjVFK8IMUmL6OdarP/ojqXjZAHI+k61Ja++w=";
+  };
+
+  vendorHash = "sha256-WcPKMF8KNx7zlsdTnFf8vnW/uZZL1F4JWqMK7+qmyCk=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/lima-vm/sshocker/pkg/version.Version=${version}"
```

Comment:
```suggestion
    "-X=github.com/lima-vm/sshocker/pkg/version.Version=${version}"
```

Reply:
Is that a new? I can't remember that we used a `=` in the past. The [manual](https://nixos.org/manual/nixpkgs/stable/#ssec-go-common-attributes) also doesn't mention a `=`.

Reply:
It's not necessary to make the package work, but it is relying on an (IMO) unintuitive behavior of nix, that converts `[ "foo" "bar baz" ]` into `foo bar baz` instead of `foo 'bar baz'`. There is a nixpkgs-hammering [lint](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/no-flags-spaces.md) that suggests something similar for `*Flags`. Using `=` here looks a little better than splitting into two elements (`"-X" "github.com...=${version}"`), so that's what I opted for, but feel free to do that instead.